Gemfile.lock

```diff
@@ -2,6 +2,7 @@ GEM
   remote: http://rubygems.org/
   specs:
     RedCloth (4.2.9)
+    RedCloth (4.2.9-x86-mingw32)
```

Comment:
Alright, I have done that in my branch. reverted the file back to the last
commit before I did work:

commit e38a18b28c0c1cc256332e796b31b5c7f7c853f6
> Author: Addy Osmani <addyosmani@gmail.com>
> Date:   Mon Apr 27 15:27:59 2015 +0100
>     Fix broken links, update stats


Have a look and see what you think. I think I have covered off all the
issues you raised.

Cheers,
Seth



[image: 3sparks llc] <contact@sethreid.co.nz>

Seth Reid | Web Consultant/Developer

0221705966 | contact@sethreid.co.nz

On 30 April 2015 at 10:39, Eddie Monge <notifications@github.com> wrote:

> In Gemfile.lock
> <https://github.com/yeoman/yeoman.io/pull/468#discussion_r29388851>:
>
> > @@ -2,6 +2,7 @@ GEM
> >    remote: http://rubygems.org/
> >    specs:
> >      RedCloth (4.2.9)
> > +    RedCloth (4.2.9-x86-mingw32)
>
> I mean this file shouldn't have the changes checked in. I don't know what
> adding those versions will do for everyone else including a build system.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/yeoman/yeoman.io/pull/468/files#r29388851>.
>